Sources/SwifterAuth.swift

```diff
@@ -116,9 +116,21 @@ public extension Swifter {
     }
     #endif
     
-    public class func handleOpenURL(_ url: URL) {
+    public class func handleOpenURL(_ url: URL, callbackURL: URL) -> Bool {
+        guard isTwitterURLScheme(url, callbackURL: callbackURL) else {
+            return false
+        }
         let notification = Notification(name: .swifterCallback, object: nil, userInfo: [CallbackNotification.optionsURLKey: url])
         NotificationCenter.default.post(notification)
+        return true;
```

Comment:
For style reasons, remove the semicolon here

Reply:
ty!